### ISSUE TITLE
Fix linting issues due to ForEachOnRange 

### ISSUE THREAD

**Issue by angrezichatterbox** [scribe-org/Scribe-Android#126]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/scribe-org/Scribe-Android/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow Scribe-Android's [Code of Conduct](https://github.com/scribe-org/Scribe-Android/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

The `ForEachOnRange` rule is currently disabled. This rule helps detect the use of `forEach` on ranges, which can lead to performance issues. Re-enabling this rule will promote better performance practices.

**Steps to see which lines have to be modified:**
1. Comment or set to true the following line in `detekt.yml`:
   ```yaml
   performance:
       ForEachOnRange:
           active: true
   ```
2. Run the command `./gradlew detekt`.
3. Review the results to identify any usages of `forEach` on ranges that need refactoring.

### Contribution

I would love to work on this issue and am more than willing to help anyone fix this issue. :)

**Comment by DeKaN** [comment]
Can I take the issue?

**Comment by angrezichatterbox** [comment]
Yes, feel free to take it up @DeKaN. Thanks :)

**Comment by andrewtavis** [comment]
Closed by #173 😊 Let us know if you find another issue you'd be interested in working on, @DeKaN!
